conda_forge_tick/update_upstream_versions.py

```diff
@@ -59,23 +59,17 @@ def get_latest_version(
 
 
 def _update_upstream_versions_sequential(
-    gx: nx.DiGraph, sources: Iterable[AbstractSource] = None,
+    _all_nodes, sources: Iterable[AbstractSource] = None,
```

Comment:
Why does this have an underscore for an arg name?

Reply:
A bad habit, sorry